test/feed-parser.test.js

```diff
@@ -25,3 +25,37 @@ test('Passing an invalid RSS feed URI should error', async () => {
 test('Passing a valid URI, but not a feed URI should error', async () => {
   await expect(feedParser('https://google.ca')).rejects.toThrow('Not a feed');
 });
+
+const assertValidFeed = (feed) => {
+  expect(Array.isArray(feed)).toBeTruthy();
+  expect(feed.length > 0).toBeTruthy();
+};
+
+
+test('Non existant feed failure case.', async () => {
+  try {
+    await feedParser('http://doesnotexists___.com');
+  } catch (err) {
+    expect(err.code).toBe('ENOTFOUND');
+  }
+});
+test('Not a feed failure case', async () => {
+  try {
+    const nonFeedURL = 'https://kerleysblog.blogspot.com';
+    await feedParser(nonFeedURL);
+  } catch (err) {
+    expect(err.code).toBe('Not a feed');
```

Comment:
Error codes have to be one of the `E--------` strings I linked you, like `ENOTFOUND`.  This won't work.